modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java

```diff
@@ -493,12 +494,22 @@ public List<CommerceOrder> getUserPendingCommerceOrders(
 			long companyId, long groupId, String keywords, int start, int end)
 		throws PortalException {
 
+		return getUserPendingCommerceOrders(
+			companyId, groupId, keywords, start, end, null);
+	}
+
+	@Override
+	public List<CommerceOrder> getUserPendingCommerceOrders(
```

Comment:
I don't like pending here. Pending is a specific status, so you can maybe call it getUserOpenCommerceOrders

Reply:
Done.